front/src/routes/integration/all/tuya/actions.js

```diff
@@ -0,0 +1,209 @@
+import debounce from 'debounce';
+import createActionsIntegration from '../../../../actions/integration';
+
+function createActions(store) {
+  const integrationActions = createActionsIntegration(store);
+  const actions = {
+    async getHouses(state) {
+      store.setState({
+        housesGetStatus: RequestStatus.Getting
+      });
+      try {
+        const params = {
+          expand: 'rooms'
+        };
+        const housesWithRooms = await state.httpClient.get(`/api/v1/house`, params);
+        store.setState({
+          housesWithRooms,
+          housesGetStatus: RequestStatus.Success
+        });
+      } catch (e) {
+        store.setState({
+          housesGetStatus: RequestStatus.Error
+        });
+      }
+    },
+
+    async getDiscoveredDevices(state) {
+      store.setState({
+        loading: true
+      });
+      try {
+        const discoveredDevices = await state.httpClient.get('/api/v1/service/tuya/discover');
+        store.setState({
+          discoveredDevices,
+          loading: false,
+          errorLoading: false
+        });
+        console.log(discoveredDevices);
+      } catch (e) {
+        store.setState({
+          loading: false,
+          errorLoading: true
+        });
+      }
+    },
+
+    async getTuyaDevices(state) {
+      store.setState({
+        getTuyaStatus: RequestStatus.Getting
+      });
+      try {
+        const options = {
+          order_dir: state.getTuyaOrderDir || 'asc'
+        };
+        if (state.tuyaSearch && state.tuyaSearch.length) {
+          options.search = state.tuyaSearch;
+        }
+
+        const tuyaDevices = await state.httpClient.get('/api/v1/service/tuya/device', options);
+        store.setState({
+          tuyaDevices,
+          getTuyaStatus: RequestStatus.Success
+        });
+      } catch (e) {
+        store.setState({
+          getTuyaStatus: e.message
+        });
+      }
+    },
+
+    updateConfiguration(state, e) {
+      const data = {};
+      data[e.target.name] = e.target.value;
+      store.setState(data);
+    },
+
+    async getTuyaConfiguration(state) {
+      let tuyaEndpoint = '';
+      let tuyaAccessKey = '';
+      let tuyaSecretKey = '';
+      let tuyaAppAccountId = '';
+
+      store.setState({
+        tuyaGetSettingsStatus: RequestStatus.Getting,
+        tuyaEndpoint,
+        tuyaAccessKey,
+        tuyaSecretKey,
+        tuyaAppAccountId
+      });
+      try {
+        const { value: endpoint } = await state.httpClient.get('/api/v1/service/tuya/variable/ENDPOINT');
+        tuyaEndpoint = endpoint;
+
+        const { value: accessKey } = await state.httpClient.get('/api/v1/service/tuya/variable/ACCESS_KEY');
+        tuyaAccessKey = accessKey;
+
+        const { value: secretKey } = await state.httpClient.get('/api/v1/service/tuya/variable/SECRET_KEY');
+        tuyaSecretKey = secretKey;
+
+        const { value: appAccountId } = await state.httpClient.get('/api/v1/service/tuya/variable/APP_ACCOUNT_UID');
```

Comment:
Can you prefix those variables with TUYA? So it's clearer in the DB that we are talking about the Tuya service?